### PR TITLE
Blog debugging app load balancers

### PR DESCRIPTION
**Pull request by chungos**
- blog on how to some what debugging application load servers
- with lots of pictures of various British comedies/comedians from the decades

### CODE REVIEW

File: content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    - cloud
+tags:
+    - AWS 
+    - how to 
+    - ALB
+author: Owen Chung 
+excerpt:
+ A multitudinous method to make a HTTP request meet a matching HTTP response.
+images: blog/neil-heavy.jpg
+thumbnail: blog/neil-heavy.jpg
```
Comment (stevemac007) [inline content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown]: mark as draft to control release

File: content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+---
+
+## Introduction
+
+From the Temples of Testers, a browser bestowed a 504 gateway timeout in your newly deployed internal facing application load balancer. There was a gnashing of molars and gurning of visages. So your application load balancer isn't responding. Don't panic and be **"oh wow! heavy heavy heavy"** like Neil above from the Young Ones.
```
Comment (stevemac007) [inline content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown]: `Neil above` am I missing a pop culture reference here?

File: content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+Be like Blackadder below with a cunning plan to debug an internal Application Load Balancer. Below are steps you too can follow to foil load balancer connectivity foibles. 
+
+![a cunning plan](/img/blog/a_cunning_plan_blackadder.jpg)
+
+
+### Can the "computer says no" defect be replicated?
+
+##### As its an internal not external load balancer
+
+![computer-says-no](/img/blog/computer_says_no.jpg)
```
Comment (stevemac007) [inline content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown]: should this image be above the second heading?

File: content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+From the Temples of Testers, a browser bestowed a 504 gateway timeout in your newly deployed internal facing application load balancer. There was a gnashing of molars and gurning of visages. So your application load balancer isn't responding. Don't panic and be **"oh wow! heavy heavy heavy"** like Neil above from the Young Ones.
+
+Be like Blackadder below with a cunning plan to debug an internal Application Load Balancer. Below are steps you too can follow to foil load balancer connectivity foibles. 
+
+![a cunning plan](/img/blog/a_cunning_plan_blackadder.jpg)
+
+
+### Can the "computer says no" defect be replicated?
+
+##### As its an internal not external load balancer
```
Comment (stevemac007) [inline content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown]: it's

File: content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+It was a [Trojan horse](https://en.wikipedia.org/wiki/Trojan_Horse) false victory. Python simple HTTP server was turned off and the correct web server was reinstated. Curl returned an invalid HTTP response again. The relevant expression for this moment rhymes with "clucking bell."
+
+![cluck-bell](/img/blog/cluck-bell.jpg)
+
+The configuration of the web server was inspected. Amazing an eyeball by the local application expert spotted the one parameter causing encryption [kerfuffles](https://en.oxforddictionaries.com/definition/kerfuffle) in the redirects of HTTP to HTTPS requests. This parameter caused requests not to be processed. As the development certificates didn't contain invalid addresses. The offending parameter was tweaked. 
+
+A curl again. The curl was harked! A HTTP 200 sprang forth from an application cradle of genesis, through a web primordial swamp, beyond an application load balancer gestalt and met its request with  its magnificent HTML payload of "login and password."
+
+**Lo! The internal application load balancer has been debugged. Computer can say "yes!"**
```
Comment (stevemac007) [inline content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown]: It's a great story - is there any summary for troubleshooting ALB's that we could close this out to leave people with a simple checklist for debugging these issues?

Comment (SianHughes) [reply content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown]: I agree with Steve - just need to make sure the fun part ties back into something practical people can take away and demonstrates your technical expertise.

File: content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown (hunk 1 above)
Comment (chungos) [reply content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown]: how do I do that?

Comment (stevemac007) [reply content/post/2018-04-30-how-to-debug-AWS-application-load-balancers.markdown]: Add `draft: true` as an attribute.

**Pull request merged**